### DISCUSSION TITLE
Standalone Electron app

### DISCUSSION THREAD

**Discussion by AlexW00** [mgmeyers/obsidian-kanban#458] [Ideas & Feature Requests]
A standalone electron app would be awesome. That way, one could use the kanban board without having to fire up obsidian every time.

**Comment by mgmeyers** [comment]
This is a great idea @AlexW00 and one I hadn't thought of. It's definitely doable, though would be quite a bit of work. I'll add this to the "someday" list, unless someone else wants to take a crack at it.
